packages/trace/src/trace.ts

```diff
@@ -43,6 +43,7 @@ export type ContextCreatedTraceEvent = {
   options: BrowserContextEventOptions,
   sdkLanguage?: Language,
   testIdAttributeName?: string,
+  baseURL?: string,
```

Comment:
This value is already written to the trace as `ContextCreatedTraceEvent.options.baseURL`, we just need to add it to the type.

Reply:
Nice! Done in https://github.com/microsoft/playwright/pull/31852/commits/c8f2d6bed6212aa2afb2612d6e5b5e1852f4d85b.